src/Riok.Mapperly/Descriptors/Mappings/QueryableProjectionMapping.cs

```diff
@@ -0,0 +1,45 @@
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using static Microsoft.CodeAnalysis.CSharp.SyntaxFactory;
+using static Riok.Mapperly.Emit.SyntaxFactoryHelper;
+
+namespace Riok.Mapperly.Descriptors.Mappings;
+
+/// <summary>
+/// A projections queryable mapping
+/// to map from one generic <see cref="IQueryable{T}"/> to another.
+/// </summary>
+public class QueryableProjectionMapping : MethodMapping
+{
+    private const string SelectLambdaParameterName = "x";
+
+    private const string QueryableReceiverName = "System.Linq.Queryable";
+    private const string SelectMethodName = nameof(Queryable.Select);
+
+    private readonly ITypeMapping _delegateMapping;
+
+    public QueryableProjectionMapping(
+        ITypeSymbol sourceType,
+        ITypeSymbol targetType,
+        ITypeMapping delegateMapping) : base(sourceType, targetType)
+    {
+        _delegateMapping = delegateMapping;
+    }
+
+    public override IEnumerable<StatementSyntax> BuildBody(TypeMappingBuildContext ctx)
+    {
+        // #nullable disable
```

Comment:
Why is this needed here? Add a comment. May also need to be added to the documentation

Reply:
I don't think just disabling nullability solves this problem correctly. What happens if values are actually null?

Don't we need to handle all nullable values correctly anyway? I don't quite get why this is needed